src/transaction/transaction_transient.cpp

```diff
@@ -126,10 +126,9 @@ tx_transient_class_registry::decache_heap_repr (const LOG_LSA &downto_lsa)
 {
   for (auto &it : m_list)
     {
-      if (it.m_last_modified_lsa > downto_lsa)
+      if (downto_lsa.is_null () || it.m_last_modified_lsa > downto_lsa)
```

Comment:
use LSA_GT instead >

Reply:
what is the reason to prefer LSA_GT to >?

Reply:
Because 2 LSAs are compared. For me, the condition should be: if (LSA_GT (it.m_last_modified_lsa, downto_lsa) .

Reply:
I prefer operator> instead of LSA_GT. it was created as syntax sugar for legacy macro's, it makes code more readable.

However, legacy macro's are heavily used and to replace them all requires a big refactoring patch. In the future we should do it, but in the meantime, we'll make sure both functions work in the same way.

Reply:
downto_lsa.is_null () seems to be useless...